docs/relational-databases/json/index-json-data.md

```diff
@@ -26,7 +26,32 @@ Database indexes improve the performance of filter and sort operations. Without
 When you store JSON data in SQL Server, typically you want to filter or sort query results by one or more *properties* of the JSON documents.  
 
 ### Example 
-In this example, assume that the AdventureWorks `SalesOrderHeader` table has an `Info` column that contains various information in JSON format about sales orders. For example, it contains information about customer, sales person, shipping and billing addresses, and so forth. You want to use values from the `Info` column to filter sales orders for a customer.
+In this example, assume that the AdventureWorks `SalesOrderHeader` table has an `Info` column that contains various information in JSON format about sales orders. For example, it contains unstructured data about customer, sales person, shipping and billing addresses, and so forth. You could use values from the `Info` column to filter sales orders for a customer. 
+
+By default, the column `Info` used does not exist, it can be created in the `AdventureWorks` database with the following code:

```

Comment:
```suggestion
By default, the column `Info` used does not exist, it can be created in the `AdventureWorks` database with the following code. Note this does not apply to the AdventureWorksLT series of sample databases.

```